Sources/Basic/Path.swift

```diff
@@ -28,14 +29,23 @@
 /// normalization, because it is normally the responsibility of the shell and
 /// not the program being invoked (e.g. when invoking `cd ~`, it is the shell
 /// that evaluates the tilde; the `cd` command receives an absolute path).
+
+import Foundation
+
+#if os(Windows)
+public let PATH_SEPARATOR = "\\"
+#else
+public let PATH_SEPARATOR = "/"
```

Comment:
I think this should be a static property on AbolsutePath so it's correctly namespaced: `AbsolutePath.pathSeparator`